easybuild/easyconfigs/d/double-down/double-down-1.1.0-foss-2022a.eb

```
@@ -0,0 +1,37 @@
+name = 'double-down'
+version = '1.1.0'
+moduleclass = 'phys'
+
+homepage = 'https://github.com/pshriwise/double-down'
+description = "double-down is a double precision interface to Embree via the Mesh Oriented dAtaBase (MOAB)."
+software_license = 'LicenseGCC'
+software_license_urls = ['https://github.com/pshriwise/double-down/blob/develop/LICENSE']
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = [{
+    'filename': 'double-down.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/pshriwise',
+        'repo_name': 'double-down',
+        'tag': f'v{version}'
+    },
+}]
+
+builddependencies = [('CMake', '3.23.1'),('make','4.3')]
+
+dependencies = [
+    ('Doxygen','1.9.4'),
+    ('Embree','3.13.4','',{'name': 'system', 'version': 'system'}),
+    ('MOAB','5.5.0'),
+    ('tbb','2021.5.0')
+]
+
+easyblock = 'CMakeMake'
+
+_moab_dir = '$EBROOTMOAB'
+configopts = f"-DMOAB_DIR={_moab_dir}"
```

We can't use fstrings on possible older versions and it's not required here anyway:
```suggestion
configopts = "-DMOAB_DIR=$EBROOTMOAB"
```

---

Ok, so as a rule easyconfigs should not contain fstrings?

---

Yes